x-pack/legacy/plugins/siem/server/lib/detection_engine/rule_actions/utils.ts

```diff
@@ -5,16 +5,27 @@
  */
 
 import { SavedObjectsUpdateResponse } from 'kibana/server';
+import { RuleAlertAction } from '../../../../common/detection_engine/types';
 import { IRuleActionsAttributesSavedObjectAttributes } from './types';
 
-export const getThrottleOptions = (throttle = 'no_actions') => ({
-  ruleThrottle: throttle,
-  alertThrottle: ['no_actions', 'rule'].includes(throttle) ? null : throttle,
+export const getThrottleOptions = (
+  throttle: string | undefined | null = 'no_actions'
+): {
+  ruleThrottle: string;
+  alertThrottle: string | null;
+} => ({
+  ruleThrottle: throttle ?? 'no_actions',
+  alertThrottle: ['no_actions', 'rule'].includes(throttle ?? 'no_actions') ? null : throttle,
```

Comment:
@patrykkopycinski look here to make sure this is correct. After I made it so the tests were able to pass down `null` by changing the non-null type assertions it looked like everywhere the `null` could be sent down and I had to change all of these to accept null.

This made it to where I had to change this function slightly to work with both `null` and `undefined`